src/main/java/study/racingCar/PlayCarRacingGame.java

```diff
@@ -0,0 +1,19 @@
+package study.racingCar;
+
+import static study.racingCar.ResultView.printRacing;
+
+public class PlayCarRacingGame {
+
+    public PlayCarRacingGame() {
+    }
+
+    public void play(int cars, CarList carList){
+        RandomNumGenerator generator = new RandomNumGenerator();
+
+        for (int j = 0; j < cars; j++) {
+            carList.getCar(j).move(generator.randomNum());
```

Comment:
만약 해당 피드백이 반영된다면 CarList에 자동차들을 관리하고 있기에 CarList를 통해 움직이도록 하면 어떨까요 :) ?

```suggestion
    public void play(int cars, CarList carList){
        RandomNumGenerator generator = new RandomNumGenerator();
        cars.move(generator);
```